pkg/alertmanager/operator.go

```diff
@@ -1790,3 +1800,14 @@ func ListOptions(name string) metav1.ListOptions {
 func tlsAssetsSecretName(name string) string {
 	return fmt.Sprintf("%s-tls-assets", prefixedName(name))
 }
+
+func GetAppliedConfigFromCondition(c *monitoringv1.Condition) *monitoringv1ac.ConditionApplyConfiguration {
+	return &monitoringv1ac.ConditionApplyConfiguration{
```

Comment:
(suggestion) let's use the helpers from the apply package.
```suggestion
	return monitoringv1ac.Condition().
		WithType(c.Type).
		WithStatus(c.Status).
		...
```

Reply:
done